migrations/versions/3c8dbc9ebf7e_more_robust_opportunities.py

```diff
@@ -25,8 +25,8 @@ def upgrade():
     sa.PrimaryKeyConstraint('id')
     )
     op.create_index(op.f('ix_opportunity_document_id'), 'opportunity_document', ['id'], unique=False)
-    op.add_column(u'opportunity', sa.Column('created_by_id', sa.Integer(), nullable=False))
-    op.add_column(u'opportunity', sa.Column('planned_advertise', sa.DateTime(), nullable=False))
+    op.add_column(u'opportunity', sa.Column('created_by_id', sa.Integer(), nullable=False, server_default=sa.schema.DefaultClause('1')))
+    op.add_column(u'opportunity', sa.Column('planned_advertise', sa.DateTime(), nullable=False, server_default=sa.schema.DefaultClause('2015-01-01 00:00:00')))
```

Comment:
@shellicious @hamhands this defaults existing opportunities as being open as of Jan 1, but there aren't any on production so I don't think it's a huge deal. Thoughts?